app/src/main/res/values/strings.xml

```diff
@@ -337,13 +340,29 @@ Please report any problems you encounter via Github.</string>
 
     <string name="export_config">Export Configuration</string>
 
+    <!-- Experimental options -->
+    <string name="use_proxy">Connect through a proxy</string>
+    <string name="do_not_use_proxy">Do not connect through a proxy</string>
+
     <string name="keep_wakelock_while_binary_running">Keep the CPU awake while Syncthing is running</string>
 
     <string name="keep_wakelock_while_binary_running_summary">Use this setting if you experience unexpected disconnects while operating on battery. This will result in increased battery consumption.</string>
 
     <string name="use_tor_title">Use Tor</string>
 
-    <string name="use_tor_summary">Force all traffic through Tor for increased privacy. Requires Orbot</string>
+    <string name="use_tor_summary">Force all traffic through Tor for increased privacy. Requires Orbot. Disables proxy options.</string>
```

Comment:
I tried this already but have to disappoint you. It's not supported on android xml to reference a concatenation of two strings. That applies both for strings.xml and app-settings.xml. So the only way implementing this would be to write concatenation code which I think is the worse alternative.